src/app/shared/components/template/components/toggle-bar/toggle-bar.html

```diff
@@ -11,12 +11,11 @@
       class="toggle_wrapper"
       [class.show-tick-cross]="params.showTickAndCross"
     >
-      <ion-toggle
-        mode="md"
-        [checked]="!!_row.value"
-        (click)="handleClick($event)"
-        legacy="true"
-      ></ion-toggle>
+      <ion-toggle 
+        mode="ios"
+        [enableOnOffLabels]="params.showTickAndCross"
```

Comment:
This should also be affected by the new `showIcons` param (see my other inline comment). In order to handle the default being to show the icons whilst supporting the legacy functionality of `show_tick_and_cross: false`, I think the logical statement you'll want here is `params.showTickAndCross && params.showIcons`. As long as both properties default to the value `true`, that should handle the logic correctly.

Reply:
I will make the default iOS because it looks better generally and it is much clearer to show the tick and cross icons.